extensions/amp-access/0.1/amp-access.js

```diff
@@ -394,6 +401,10 @@ export class AccessService {
       this.toggleTopClass_('amp-access-loading', false);
       this.toggleTopClass_('amp-access-error', true);
     });
+    // At least one promise should succeed.
```

Comment:
Updated comment. The right wording "first promise is always required to complete".